equinox/nn/_embedding.py

```diff
@@ -220,12 +225,16 @@ def __call__(
                 freqs_cis = internal_rope_embedding_cache[embedding_size]
                 freqs_cis_seq_len, _ = freqs_cis.shape
                 if seq_len > freqs_cis_seq_len:
-                    freqs_cis = self.precompute_freqs_cis(embedding_size, seq_len)
+                    freqs_cis = self.precompute_freqs_cis(
+                        embedding_size, seq_len, self.theta
```

Comment:
Nit: let's remove the default argument for `precompute_freqs_cis`, I don't think it's ever used now?

Reply:
Yep!